genny/test_utils.go

```diff
@@ -0,0 +1,16 @@
+package genny
+
+import (
+	"strings"
+)
+
+/**
+ * Source: https://www.programming-books.io/essential/go/normalize-newlines-1d3abcf6f17c4186bb9617fa14074e48
+ */
+func NormalizeNewlines(d string) string {
```

Comment:
This isn’t in a test file, so it is a part of the public API. Please move it to a test file.